cmd/age/encrypted_keys.go

```diff
@@ -119,8 +127,7 @@ func readPassphrase(prompt string) ([]byte, error) {
 			return nil, err
 		}
 		defer out.Close()
-	} else if _, err := os.Stat("/dev/tty"); err == nil {
-		tty, err := os.OpenFile("/dev/tty", os.O_RDWR, 0)
+	} else if tty, err := openTTY(); err == nil {
```

Comment:
```suggestion
	} else if tty, err := os.OpenFile("/dev/tty", os.O_RDWR, 0); err == nil {
```

No need for the Stat at all, right?

Reply:
Totally agree, good catch, thank you!